hapi-fees/src/test/java/com/hedera/services/pricing/BaseOperationUsageTest.java

```diff
@@ -79,6 +79,9 @@ void picksAppropriateCryptoOp() {
 
 		mock.baseUsageFor(HederaFunctionality.CryptoApproveAllowance, DEFAULT);
 		verify(mock).cryptoUpdate(0);
+
+		mock.baseUsageFor(HederaFunctionality.CryptoAdjustAllowance, DEFAULT);
+		verify(mock).cryptoUpdate(0);
```

Comment:
The two `verify`'s above are supposed to be for the `crypto{Approve,Adjust}Allowance` methods, right?

Reply:
Yes .. 🤦‍♀️  . Fixed it ! Thank you!